ansible/roles/fanout/templates/sonic_deploy_202012.j2

```diff
@@ -61,6 +61,12 @@
             "gwaddr": "{{ device_info[inventory_hostname]["ManagementGw"] }}"
         }
     },
+    "MGMT_PORT": {
```

Comment:
Should we add this for 202012?

Reply:
it's not necessary, but I tested and it shows no error.